core/runtime/src/main/java/io/quarkus/runtime/logging/FileConfig.java

```diff
@@ -49,4 +49,61 @@
     @ConfigItem(defaultValue = "quarkus.log")
     File path;
 
+    /**
+     * File rotating config
+     */
+    RotatingConfig rotating;
+
+    @ConfigGroup
+    public static class RotatingConfig {
+        /**
+         * File handler rotating type
+         */
+        @ConfigItem(defaultValue = "none")
+        RotatingType type;
+
+        /**
+         * File handler rotating size
+         */
+        @ConfigItem(defaultValue = "0")
+        Long rotationSize;
+
+        /**
+         * Max backup index
+         */
+        @ConfigItem(defaultValue = "1")
+        Integer maxBackupIndex;
+
+        /**
+         * File handler rotating suffix
+         */
+        @ConfigItem(defaultValue = "-yyyy-MM-dd")
+        String suffix;
+
+        /**
+         * File handler rotating on boot
+         */
+        @ConfigItem(defaultValue = "true")
+        boolean rotateOnBoot;
+    }
+
+    public enum RotatingType {
+        NONE,
+        SIZE_ROTATING,
+        PERIODIC_ROTATING,
+        PERIODIC_SIZE_ROTATION;
```

Comment:
You have ROTATION here whereas the others are called ROTATING.

Reply:
Good catch.